.travis.yml

```diff
@@ -18,7 +18,9 @@ install:
 - python setup.py develop
 - pip install .[test]
 - pip install tensorflow xgboost matplotlib ipython codecov torch torchvision
-script: python -m pytest .
+script:
+  - python -m pytest tests
+  - pylint tests/explainers/test_tree.py
```

Comment:
Has this been left in accidentally? I guess we would either lint the whole project as part of the CI or none of it, rather than just this one file

Reply:
I spent about 3 hours conforming this one file, because I will be working on it. It's just not possible to do the entire code base in one go, It's too much work and the diff is too large.

Reply:
I wasn't suggesting you reformat the entire code base, don't worry! One thing I would say though is that there may be better ways to contribute than to reformat all of the test files. Ultimately formatting is a choice and can often lead to a lot of debate, but doesn't add too much value to the code base one way or another. 

I think adding test skips is definitely worthwhile, and moving imports too, but other than that it is just another potential source of a bug creeping into code which already works.